### DISCUSSION TITLE
MJML an MJ-TEXT problem

### DISCUSSION THREAD

**Discussion by pepoteloko** [GrapesJS/grapesjs#5438] [Q&A] [answered]
We use GrapesJS with MJML plugin to design our emails.

We have seen that, in the first load of the page our mj-text, become line-selectable. I can select any of the lines of the text, i can even select an li element.

And after editing the text, it becomes other time a one big unique mj-text object. If I add a new text, i cannot select the LI items individually. But when I edit the template again, on the first load, we can select any small items inside mj-text.

This is not a problem unless you use the move button and move a P outside of the mj-text :)

Any ideas of where is the problem?

I've try to stop using P inside the mj-text, but the problem still exists.
I am using CKEditor

**Comment by artf** [comment, marked as answer]
Hi @pepoteloko did you try to use `disableTextInnerChilds` option, added in [this version](https://github.com/GrapesJS/grapesjs/releases/tag/v0.21.2)

**Reply by wsaca** [reply]
Hi @artf, is there a way to prevent children of `mj-text` from being created?

In the following example new components (default and text) are created when I edit the content, I'm using the option you specified.

```html
<div style="font-family: Ubuntu, Helvetica, Arial, sans-serif; font-size: 16px; font-weight: bold; letter-spacing: 1px; line-height: 22px; text-align: center; text-transform: uppercase; color: rgb(255, 255, 255); pointer-events: all;" contenteditable="true">
  Austin, TX
  <br id="irxmfs" data-gjs-type="default" draggable="true" class="gjs-no-pointer">
  <u id="iq34wk" data-gjs-type="text" draggable="true" class="gjs-no-pointer">-</u>
  <br id="i1pxkf" data-gjs-type="default" draggable="true" class="gjs-no-pointer">
</div>
```

**Comment by pepoteloko** [comment]
Thanks a lot four your help.
Now we only got big mj-text blocks :)
